### DISCUSSION TITLE
Progress bar should print to stderr

### DISCUSSION THREAD

**Discussion by Quarlos** [MilesCranmer/PySR#310] [Ideas]
I see that the progress bar can be turned off by setting progress=False. However, sometimes it is useful to look at the progress, while at the same time piping the results to another process. It looks like `SymbolicRegression.jl` is using an older version of `ProgressBars.jl`. 

It has been pointed out that default behaviour of `ProgressBars.jl` should be to print to stderr - [cloud-oak/ProgressBars.jl/pull/47](https://github.com/cloud-oak/ProgressBars.jl/pull/47) -, and `ProgressBars.jl` has been updated to behave that way [cloud-oak/ProgressBars.jl](https://github.com/cloud-oak/ProgressBars.jl). Is there an easy way of making `SymbolicRegression.jl` compatible with updated `ProgressBars.jl`?

**Comment by MilesCranmer** [comment]
Good idea. The file `src/ProgressBars.jl` should be pretty much the same as the main repository except for a few tweaks. The patch from that PR should be hopefully straightforward to implement - want to add it in a pull request?

**Reply by Quarlos** [reply]
Sure, will give it a try. Still not really familiar with julia package workflow and language so this might take awhile.

**Comment by MilesCranmer** [comment]
@Quarlos once #157 is merged, the upstream ProgressBars.jl will be used.

**Reply by Quarlos** [reply]
Nice, thanks, @MilesCranmer!
